.travis.yml

```diff
@@ -86,6 +86,9 @@ jobs:
     name: "Tests"
     script:
     - tar xzf $STACK_WORK_CACHE
+    - travis_retry curl -L -o hermes-testnet.tar.gz https://github.com/input-output-hk/cardano-wallet/raw/data-integration-testing/hermes-testnet.tar.gz
```

Comment:
why it is called hermes? ;-)

Reply:
No idea. Ask the rust team :sweat_smile: